examples/widgets/scale/index.rst

```diff
@@ -34,3 +34,9 @@ A round scale with multiple needles, resembling a clock
 .. lv_example:: widgets/scale/lv_example_scale_6
   :language: c
 
+Customizing scale major tick labels with `LV_EVENT_DRAW_TASK_ADDED` event
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""
```

Comment:
Length needs to be consistent

Reply:
See 538bc87f3

Reply:
The decoration under the title should have the same amount of characters as the title.

Reply:
Sorry, I'm new to this documentation format, see cd8cbd7ce